install_dependencies.sh

```diff
@@ -7,7 +7,7 @@ go get -u -v github.com/livepeer/m3u8
 go get -u -v go.opencensus.io/stats
 go get -u -v go.opencensus.io/tag
 go get -u google.golang.org/grpc
-go get github.com/alecthomas/gometalinter && gometalinter --install
+go get github.com/alecthomas/gometalinter #&& gometalinter --install
```

Comment:
did something change with `gometalinter`?

Reply:
nop, I don't remember why I commented this out. sorry 🙏